syncd/syncd.cpp

```diff
@@ -3228,6 +3310,7 @@ int syncd_main(int argc, char **argv)
 
     std::shared_ptr<swss::ConsumerTable> asicState = std::make_shared<swss::ConsumerTable>(dbAsic.get(), ASIC_STATE_TABLE);
     std::shared_ptr<swss::NotificationConsumer> restartQuery = std::make_shared<swss::NotificationConsumer>(dbAsic.get(), "RESTARTQUERY");
+    std::shared_ptr<swss::NotificationConsumer> flushfdbRequest = std::make_shared<swss::NotificationConsumer>(dbAsic.get(), "FLUSHFDBREQUEST");
```

Comment:
flush fdb request should be initiated via SAI api, and IMO it should be synchronous call similar to GET, and called as GET api, no need for another handler

Reply:
When started the design my idea is that since "clear fdb" is a one-shot action, not like a normal configuration, should not change configurations in the DB, so implement it via publish/subscribe mechanism is a quite straightforward way.   Make it synchronous is a good suggestion, I am thinking next phase when adding "flush port" and "flush vlan" support,  also make cli utilities to wait for a response from syncd or somewhere.

Reply:
I agree with @kcudnik , the request should be initiated via SAI API. the sonic-clear command should invoke APP DB and let orchagent to issue corresponding SAI API.

Reply:
@keboliu, you probably did not understand "synchronous call similiar to GET api". You probably need to look at how GET api are implemented in syncd and sairedis.

Reply:
@lguohan yes, can you give some example for GET api if you have now?  And for your previous comments "regarding invoke APP DB and let orchagent to call SAI API", do you mean we need to add some new configuration in the APP DB and trigger orchagent with manipulating this new configuration?